system-test/fixtures/sample/src/index.js

```diff
@@ -17,7 +17,7 @@
 // ** All changes to this file may be overwritten. **
 
 /* eslint-disable node/no-missing-require, no-unused-vars */
-const managedidentities = require('@google-cloud/managed-identities');
+const managedidentities = require('managedidentities');
```

Comment:
@alexander-fenster @sofisl  I am seeing this happen across a bunch of our newer repositories, but I can't find the glitch - getting too removed from code day to day :). What's up here?